test/test-tty.c

```diff
@@ -28,6 +28,14 @@
 #else /*  Unix */
 # include <fcntl.h>
 # include <unistd.h>
+# if defined(__linux__)
+#  include <pty.h>
+# elif defined(__OpenBSD__) || defined(__NetBSD__) || defined(__APPLE__)
+#  include <util.h>
+# elif defined(__FreeBSD__) || defined(__DragonFly__)
+#  include <libutil.h>
+# endif
+# include "unix/internal.h"
```

Comment:
this is not going to fly. Use the value of UV_STREAM_BLOCKING directly and leave a comment. The OSX bot fails because of this.